style.css

```
@@ -66,4 +66,37 @@
 
 .disabled {
   cursor:not-allowed;
-}
\ No newline at end of file
+}
+
+.fit-content {
+  max-width: fit-content;
+}
+
+.embedded-safe {
+  position:static;
```

```suggestion
  position: static;
```

És ha jól értem akkor ugyanez kell a .embedded-hez is és akkor nem kell ismételni stílust pár sorral lejebb megint